projects/rocprofiler-compute/CMakeLists.txt

```diff
@@ -170,13 +170,17 @@ endif()
 message(STATUS "Enable tests compilation: ${ENABLE_TESTS}")
 
 enable_testing()
+include(CTest)
 
 option(ENABLE_COVERAGE "Enable code coverage" OFF)
```

Comment:
A minor thing we could improve later: we might want to add prefix like ROCPROF_COMPUTE_ENABLE_COVERAGE for rocprof-compute specific option. As you might know, all what we defined here can be used as higher level project (unless it is common def).